donut/modules/marketplace/helpers.py

```diff
@@ -768,13 +1155,15 @@ def get_textbook_info_from_textbook_id(textbook_id):
     Returns:
         result: A list of the textbook title and author.
     """
-    query = sqlalchemy.text(
-        """SELECT textbook_title, textbook_author FROM marketplace_textbooks WHERE textbook_id=:textbook_id"""
-    )
-    result = flask.g.db.execute(query, textbook_id=textbook_id).first()
+    s = 'SELECT textbook_title, textbook_author FROM marketplace_textbooks WHERE textbook_id=%s'
+
+    with flask.g.pymysql_db.cursor() as cursor:
+        cursor.execute(s, [textbook_id])
+        result = cursor.fetchone()
+
     if result == None:
         return None
-    return list(result)
+    return [result['textbook_title'], result['textbook_author']]
```

Comment:
Tuples allow for better optimization:
```suggestion
    return (result['textbook_title'], result['textbook_author'])
```